package.json

```diff
@@ -17,6 +17,31 @@
     "resetMocks": false,
     "setupFiles": [
       "jest-localstorage-mock"
+    ],
+    "start": "sass --watch src/scss:dist/css & live-server",
```

Comment:
These commands are in the wrong place. They should be under "scripts"

Reply:
I moved all these up into scripts